apps/www/components/BuiltWithSupabase/index.tsx

```diff
@@ -16,7 +16,7 @@ const BuiltExamples = () => {
         <p className="p">There are many example apps and starter projects to get you started.</p>
         <div className="flex justify-center gap-2 py-4">
           <Link href="/docs/guides/examples" as="/docs/guides/examples" passHref>
-            <a>
+            <a tabIndex={-1}>
```

Comment:
you can do 

```
<Link pashref>
  <Button aschild>
    <a>

Reply:
Oh, that's how it's composed... I was trying the Button outside of the Link tag and it wasn't working 😁
Will update thanks 👍